### PR TITLE
Load DriveItem icons from filesystem

### PR DESCRIPTION
**Pull request by duke7553**
**Details of Changes**
This changed behavior will fallback to getting the old icons when necessary.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

**Comment by yaichenbaum** [comment]
> This changed behavior will fallback to getting the old icons when necessary.

@duke7553 Which old icons? I assume the Segoe assets, can you confirm?

File: Files/Filesystem/DriveItem.cs [hunk 1]
```diff
@@ -74,7 +75,10 @@ public DriveType Type
             set
             {
                 type = value;
-                SetGlyph(type);
+                if (Icon == null)
+                {
+                    SetGlyph(type);
+                }
```
Comment (lukeblevins) [inline Files/Filesystem/DriveItem.cs:81]: @yair100

Comment (duke7553) [reply Files/Filesystem/DriveItem.cs:81]: For now, it should fallback to the previous SetGlyph behavior

Comment (yaichenbaum) [reply Files/Filesystem/DriveItem.cs:81]: It doesn't look like the fallback is doing anything on 20H1.

Comment (lukeblevins) [reply Files/Filesystem/DriveItem.cs:81]: @yair100 Can you elaborate because that could mean a number of things?

Comment (yaichenbaum) [reply Files/Filesystem/DriveItem.cs:81]: Shouldn't I see the Segoe glyphs instead of the icons from File Explorer on 20H1?

Comment (lukeblevins) [reply Files/Filesystem/DriveItem.cs:81]: @yair100 That's not included in this PR.

Comment (yaichenbaum) [reply Files/Filesystem/DriveItem.cs:81]: Wasn't the previous SetGlyph behavior to use the segoe assets?

Comment (yaichenbaum) [reply Files/Filesystem/DriveItem.cs:81]: @duke7553 When is it supposed to fallback to that?

**Pull request merged**